packages/@aws-cdk/aws-ecs-patterns/package-lock.json

```diff
@@ -0,0 +1,2990 @@
+{
+  "name": "@aws-cdk/aws-ecs-patterns",
+  "version": "0.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "@aws-cdk/assets": {
```

Comment:
Can you please delete the `package-lock.json` file. The yarn-lock keeps package dependencies in sync so we don't need to commit `package-lock.json` files for individual modules.

Reply:
Thank you!
I deleted `package-lock.json`.